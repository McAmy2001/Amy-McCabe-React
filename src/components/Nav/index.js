import React, { useState } from 'react'
 
function Nav(props) {
  console.log(props);
  console.log(`${props.currentContent === 'About'}`);
  return(
    <nav className='nav'>
        <ul>
          <li id='About' className={`nav-list-item ${props.currentContent === 'About' && 'navActive'}`}>
          <button className={`${props.currentContent === 'About' && 'navActive'}`} onClick={() => props.setCurrentContent('About')}>About Me</button>
          </li>
          <li id='Portfolio' className='nav-list-item'>
            <button onClick={() => props.setCurrentContent('Portfolio')}>Portfolio</button>
          </li>
          <li id='Contact' className='nav-list-item'> 
          <button onClick={() => props.setCurrentContent('Contact')}>Contact Me</button>
          </li>
          <li id='Resume' className='nav-list-item'>
          <button onClick={() => props.setCurrentContent('Resume')}>Resume</button>
          </li>
        </ul>
      </nav>
  )
}



export default Nav;