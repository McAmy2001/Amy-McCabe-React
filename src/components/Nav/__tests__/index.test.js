import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Nav from '..';

afterEach(cleanup);

describe('Nav component', () => {
  it('renders', () => {
    render(<Nav></Nav>)
  });
  it('matches snapshot', () => {
    const { asFragment } = render(<Nav></Nav>);
    expect(asFragment()).toMatchSnapshot();
  });
})