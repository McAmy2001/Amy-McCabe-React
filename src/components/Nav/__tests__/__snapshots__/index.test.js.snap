// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches snapshot 1`] = `
<DocumentFragment>
  <nav>
    <ul>
      <li>
        <a
          href="#about"
        >
          About Me
        </a>
      </li>
      <li>
        <a
          href="#portfolio"
        >
          Portfolio
        </a>
      </li>
      <li>
        Contact
      </li>
      <li>
        Resume
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
